test/interactive-remove.sh

```diff
@@ -2,30 +2,30 @@
 
 . $(dirname "${BASH_SOURCE[0]}")/common.sh
 
-$DUB fetch dub --version=0.9.20 && [ -d $HOME/.dub/packages/dub-0.9.20/dub ]
-$DUB fetch dub --version=0.9.21 && [ -d $HOME/.dub/packages/dub-0.9.21/dub ]
+$DUB fetch dub --version=1.9.0 && [ -d $HOME/.dub/packages/dub-1.9.0/dub ]
+$DUB fetch dub --version=1.10.0 && [ -d $HOME/.dub/packages/dub-1.10.0/dub ]
 if $DUB remove dub --non-interactive 2>/dev/null; then
     die $LINENO 'Non-interactive remove should fail'
 fi
-echo 1 | $DUB remove dub | tr -d '\n' | grep --ignore-case 'select.*0\.9\.20.*0\.9\.21.*'
-if [ -d $HOME/.dub/packages/dub-0.9.20/dub ]; then
-    die $LINENO 'Failed to remove dub-0.9.20'
+echo 1 | $DUB remove dub | tr -d '\n' | grep --ignore-case 'select.*1\.9\.0.*1\.10\.0.*'
```

Comment:
Hmm this doesn't seem to work with DMD 2.077:

```
Select version of 'dub' to remove from location 'user':1) 1.1.02) 1.9.03) 1.10.04) all versions> Removing dub in /home/travis/.dub/packages/dub-1.1.0/dub/Removed package: 'dub'Removed dub, version 1.1.0.
[ERROR] /home/travis/build/dlang/dub/test/interactive-remove.sh:12 Failed to remove dub-1.9.0
```

Reply:
Ouch this is probably due to COVERAGE mode and dub's testsuite being rerun multiple times.